install/debug.conf.template

```diff
@@ -238,6 +238,20 @@ ModPagespeedLoadFromFile "http://@@APACHE_DOMAIN@@/mod_pagespeed_test/ipro/insta
   Header append 'Cache-Control' 'no-transform'
 </Directory>
 
+<Directory "@@APACHE_DOC_ROOT@@/mod_pagespeed_test/strip_subresource_hints/default" >
+  ModPagespeedDisableFilters add_instrumentation
+</Directory>
+
+<Directory "@@APACHE_DOC_ROOT@@/mod_pagespeed_test/strip_subresource_hints/preserve_on/" >
+  ModPagespeedPreserveSubresourceHints on
+  ModPagespeedDisableFilters add_instrumentation
+</Directory>
+
+<Directory "@@APACHE_DOC_ROOT@@/mod_pagespeed_test/strip_subresource_hints/preserve_off/" >
+  ModPagespeedPreserveSubresourceHints off
+  ModPagespeedDisableFilters add_instrumentation
+</Directory>
+
```

Comment:
Can we add one with disallow?  If we have:

    ModPagespeedDisallow "*/loaded_from_js/*.js"

then we shouldn't strip subresource hints for matching files.